### DISCUSSION TITLE
Inaccurate translation when using Swedish language

### DISCUSSION THREAD

**Discussion by Nojahhh** [cockpit-project/cockpit#18091] [General]
Hello and thank you for a great software!

I know this might seem silly to some but since Cockpit seem to use direct translation using google translate (or similar) some information and titles get inaccurately translated. One example is 'Update History' which should display 'Uppdateringshistorik' but is now displaying 'Uppdatera historiken' which basically means 'Update the history' and that might cause confusion with some who are new to the software, etc..

Not entirely sure which forum I should post this in but hopefully this could be useful for future updates.

**Comment by marusak** [comment]
We do not use google translate nor anything similar. We depend on real people translating it. It happens here: https://translate.fedoraproject.org/projects/cockpit/main/sv/

That being said, it is possible that it was translated by someone who translated it without any context or just misunderstood the usage. I have also heard that couple years ago someone tried using google-translate-like tools to do the translations so this also might be some fallback from it.

However, we depend on upstream help of translators so if you see any issues and you would like to help please log into Weblate and help us make Cockpit better :) Thanks!

**Reply by Nojahhh** [reply]
I just wanted to bump this as I have suggested a few translations to your Weblate but it doesn't seem like anyone is looking in to them. It's closing in on a year since I made my first suggestion. I've seen some Weblate updates in your commit history but none of the suggestions have been accepted or implemeted yet. One reason might be since I gave the suggestions as an "anonymous" user. Anyways, thought this might help!

**Reply by marusak** [reply]
Please don't suggest but rather just commit them directly. There is no dedicated person to go through and accept changes. Thanks!
